### PR TITLE
Load OMIM from files

### PR DESCRIPTION
**Pull request by moonso**
This PR adds the functionality to load and update OMIM-AUTO without an omim api-key by using files instead

**How to test**:
1. Instal branch
1. run `scout load panel --omim --genemap2 path/to/file --mim2genes path/to/file`

**Expected outcome**:
The load functionality should work without api-key

**Review:**
- [x] code approved by @rasi
- [x] tests executed by @moonso

### CODE REVIEW

File: scout/adapter/mongo/panel.py [hunk 1]
```diff
@@ -48,8 +43,15 @@ def load_panel(self, parsed_panel):
...
-    def load_omim_panel(self, api_key, institute=None):
-        """Create and load the OMIM-AUTO panel"""
+    def load_omim_panel(self, genemap2_lines, mim2gene_lines, institute=None):
+        """Create and load the OMIM-AUTO panel
+
+        If the panel already exists, update with new information and increase version
+
+        Args:
+            genemap_lines(iterable(str)): The genemap2 file information
+            mim2gene_lines(iterable(str)): The mim2genes file information
```
Comment (northwestwitch) [inline scout/adapter/mongo/panel.py]: Here you could add a comment for institute param

File: scout/adapter/mongo/panel.py [hunk 2]
```diff
@@ -61,16 +63,13 @@ def load_omim_panel(self, api_key, institute=None):
...
-            mim_files = fetch_mim_files(api_key=api_key, genemap2=True, mim2genes=True)
-        except Exception as err:
-            raise err
-
         date_string = None
         # Get the correct date when omim files where released
-        for line in mim_files["genemap2"]:
+        for line in genemap2_lines:
             if "Generated" in line:
                 date_string = line.split(":")[-1].lstrip().rstrip()
```
Comment (northwestwitch) [inline scout/adapter/mongo/panel.py]: lstrip().rstrip() isn't the same as just strip()?

File: scout/commands/load/panel.py [hunk 3]
```diff
@@ -35,10 +30,23 @@
...
+    type=click.Path(exists=True),
+    help="Path to file in omim mim2genes format",
+)
 @click.option(
     "--omim",
     is_flag=True,
-    help="Load the OMIM-AUTO panel into scout. A OMIM api key is required to do this(https://omim.org/api).",
+    help=(
+        "Load the OMIM-AUTO panel into scout."
+        " A OMIM api key is required to do this(https://omim.org/api)."
```
Comment (northwestwitch) [inline scout/commands/load/panel.py]:
```suggestion
        " AN OMIM api key is required to do this(https://omim.org/api)."
```
```suggestion
        " A OMIM api key is required to do this(https://omim.org/api)."
```

File: scout/parse/panel.py [hunk 4]
```diff
@@ -196,63 +194,66 @@ def parse_genes(gene_lines):
...
-                line_length = 0
-                for alt in delimiters:
-                    head_line = line.split(alt)
-                    if len(head_line) > line_length:
-                        line_length = len(head_line)
-                        delimiter = alt
-
-                if "hgnc" in line or "HGNC" in line:
-                    header = [word.lower() for word in line.split(delimiter)]
+            if line.startswith("##"):
```
Comment (northwestwitch) [inline scout/parse/panel.py]: Maybe this could be moved before `if line.startswith("#"):`?

File: scout/parse/panel.py [hunk 5]
```diff
@@ -196,63 +194,66 @@ def parse_genes(gene_lines):
...
+                continue
```
Comment (northwestwitch) [inline scout/parse/panel.py]: 👍 I like the simplified code here!

File: scout/adapter/mongo/panel.py (hunk 2 above)
Comment (moonso) [reply scout/adapter/mongo/panel.py]: Haha yes I don't understand why not strip is used..

File: scout/adapter/mongo/panel.py (hunk 1 above)
Comment (moonso) [reply scout/adapter/mongo/panel.py]: Fixed!

File: scout/adapter/mongo/panel.py (hunk 2 above)
Comment (moonso) [reply scout/adapter/mongo/panel.py]: Fixed!

File: scout/parse/panel.py (hunk 4 above)
Comment (moonso) [reply scout/parse/panel.py]: Fixed!

**Comment by moonso** [comment]
Ok @northwestwitch I'm done and done

**Pull request merged**